mycity/platforms/amazon/lambda/custom/lambda_function.py

```diff
@@ -48,8 +48,18 @@ def platform_to_mycity_request(event):
     mycity_request.request_id = event['request']['requestId']
     mycity_request.is_new_session = event['session']['new']
     mycity_request.session_id = event['session']['sessionId']
-    mycity_request.device_id = event['context']['System']['device']['deviceId']
-    mycity_request.api_access_token = event['context']['System']['apiAccessToken']
+
+    try:
+        mycity_request.device_id = event['context']['System']['device']['deviceId']
+    except KeyError:

```

Comment:
Maybe using `get` with a default argument is more elegant?

https://wiki.python.org/moin/KeyError